README.md

```diff
@@ -56,10 +56,15 @@ You have to ensure that your local environment has the following:
 - Clone this repository.
 - Ensure the `JAVA_HOME` environment variable your terminal is configured to point to JDK11+.
 - Compile the project using `build/sbt package`
+- Install `docker`
 
 ### Run the UC Server
-In a terminal, in the cloned repository root directory, start the UC server.
+In a terminal, in the cloned repository root directory, start Mysql server.
+```sh
+docker-compose -f etc/db/docker-compose.yml up -d
+```
```

Comment:
why not put this line inside `bin/start-uc-server`?

Reply:
It'll get more difficult if we end up supporting multiple types of database so I didn't want the server startup to be dependent on the db setup.